app/controllers/stripe-setup/add-psp-account-details/get-controller.test.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
+
+// NPM dependencies
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+const sinon = require('sinon')
+const proxyquire = require('proxyquire')
+const paths = require('../../../paths')
+
+// Global setup
+chai.use(chaiAsPromised)
+const { expect } = chai
+
+const getController = function getController (stripeAccountSetupResponse) {
+  return proxyquire('./get-controller', {
+    '../../../services/clients/connector_client': {
+      ConnectorClient: function () {
+        this.getStripeAccountSetup = (gatewayAccountId, correlationId) => {
+          return new Promise(resolve => {
+            resolve(stripeAccountSetupResponse)
+          })
+        }
+      }
+    }
+  })
+}
+
+describe('get controller', () => {
+  const req = {
+    account: {
+      gateway_account_id: 'gatewayId'
+    },
+    correlationId: 'requestId'
+  }
+
+  let res
+
+  beforeEach(() => {
+    res = {
+      setHeader: sinon.stub(),
+      status: sinon.spy(),
+      redirect: sinon.spy(),
+      render: sinon.spy()
+    }
+  })
+
+  it('should redirect to bank account setup page', async () => {
+    const controller = getController({
+      bankAccount: false,
+      responsiblePerson: false,
+      vatNumberCompanyNumber: false
+    })
+    await controller(req, res)
+    expect(res.redirect.calledWith(303, paths.stripeSetup.bankDetails)).to.equal(true)
```

Comment:
I've found that `sinon.assert` is useful here as when it passes/ fails it will tell you the exact difference between what was called with and what was expected

```
sinon.assert.calledWith(spy, 303, paths.stripeSetup.bankDetails)
```

With the current method it will just tell you that it expected `calledWith` to be `true` but was `false`.

Reply:
good tip, have found the output was lacking